src/components/Block.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="block-wrapper" :style="{ background: background }" :class="{ bordered: bordered}"> 
```

Comment:
{ background: background } can be replaced by { background }, same with { bordered: bordered} -> { bordered }